src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/runner/DeletedAccountsApiTest.java

```diff
@@ -80,5 +88,95 @@ public void assertHttpForbiddenWithInvalidS2SToken() throws JsonProcessingExcept
         );
     }
 
+    @Test
+    @Title("Assert response code of 201 for GET Request deletedAccounts")
```

Comment:
It shouldn't change the outcome of this test, but this should be 200, not 201. (201 is CREATED, 200 is OK)